controller/src/main/java/io/pravega/controller/server/rest/generated/api/ApiException.java

```diff
@@ -1,6 +1,6 @@
 package io.pravega.controller.server.rest.generated.api;
 
-@SuppressWarnings("all")
```

Comment:
revert this please. here and everywhere else

Reply:
Done